src/DispatchFunctional.jl

```diff
@@ -12,8 +12,8 @@ function LibxcFunctional(identifier::Symbol)
     @assert fun.kind   in (:exchange, :correlation, :exchange_correlation)
     kind = Dict(:exchange => :x, :correlation => :c, :exchange_correlation => :xc)[fun.kind]
 
-    @assert fun.family in (:lda, :gga, :mgga)  # Hybrids not supported yet.
-    if fun.family == :mgga && Libxc.needs_laplacian(fun)
+    @assert fun.family in (:lda, :hyb_lda, :gga, :hyb_gga, :mgga, :hyb_mgga)
+    if (fun.family == :mgga || fun.family == :hyb_mgga) && Libxc.needs_laplacian(fun)
         family = :mggal
```

Comment:
Or `:hyb_mggal` actually.

Reply:
Actually see my comments in https://github.com/JuliaMolSim/DftFunctionals.jl/pull/8/.